### ISSUE TITLE
OACMPCTF1 strange shadows

### ISSUE THREAD

**Issue by The-Gig** [NeonKnightOA/oacmpvol1#20]
I'm checking old "todo lists" from the forum against z_oacmp1v4_beta-220322.pk3.

OACMPCTF1: Noticed a few "shadow-like" artifacts in some places. Maybe it's just that the lightning compiling works a bit different with patches and regular brushes and there may be little to do about it. I don't know, however it does not seem a major issue.
![shot0235](https://user-images.githubusercontent.com/23258514/161803009-6dc59ed5-5fc4-4af1-9f5f-ba021f667c23.jpg)
![shot0237](https://user-images.githubusercontent.com/23258514/161803016-bbcf46aa-213a-457d-9c64-da8a2a15076f.jpg)

**Comment by NeonKnightOA** [comment]
I'll try some solutions I have in mind.

**Comment by EddieBrrrock** [comment]
same at the bottom :
![shot0028](https://user-images.githubusercontent.com/99912467/162947066-7b60b862-59ea-41f2-9fe1-f05a1f9decbf.jpg)
![shot0029](https://user-images.githubusercontent.com/99912467/162947088-d0d248e5-ddf1-439a-bb4e-504e68cf2bf5.jpg)

**Comment by NeonKnightOA** [comment]
It's going to be a bit hard to solve this one considering the skybox used. But I'll try to do my best.

I hate the complete inability to preview realtime lighting on the usual editors. Guess I was too used to UnrealEd allowing me to see lighting in the Editor and the game being one and the same. 🤷‍♂️

**Comment by NeonKnightOA** [comment]
Yeah, I love Jan's maps, but not really a fan of the "old school" construction style.

**Comment by The-Gig** [comment]
I think NetRadiant Custom had got an AutoCaulk feature which sets common/caulk to all brush faces which are completely unreachable by the player. I haven't really tested it on a real map yet.

**Comment by NeonKnightOA** [comment]
Really? I need to test that!

**Comment by NeonKnightOA** [comment]
https://mega.nz/file/bMZkRJBa#J_rUm2nz2SpiMFl6kBY0eMceeCLOJ1PLfnz_-CQRzCw
New test version. AutoCaulk was applied to the map, I don't think I saw many shadows, but I'd rather be sure by letting others test it and confirm if the glitches are gone. Perhaps I was too tired to notice them.

**Comment by NeonKnightOA** [comment]
No reports in over a year, closing.

**Comment by The-Gig** [comment]
Sorry for the very late reply. I tested version 230430.
Maybe it looks a bit better now, but if one watches closely, there are still some issues.
![shot0076](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/f12d76c2-8b52-4cf3-a4ce-550063413b1c)
![shot0079](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/8726b261-063e-4c6d-98b0-96b9bc76e709)
![shot0080](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/db67f11c-f55a-47b3-b6db-3d7671fdcb55)
![shot0078](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/094c9141-3ed0-4501-8717-293d37ef60fc)
Including this one:
![shot0077](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/3240c319-aaa0-4fc8-b0eb-751d7e764d52)
Wondering whether placing the top of the brushes/patches at the same height might help, as I theorized here <https://github.com/NeonKnightOA/oacmpvol1/issues/20#issuecomment-1468235307>

**Comment by The-Gig** [comment]
I've made a test version with the fix OACPCTF1_GIGTEST2 (2024-05-24), where I fixed the issue:
[oacmpctf1_gigtest2 (20240524).zip](https://github.com/NeonKnightOA/oacmpvol1/files/15430719/oacmpctf1_gigtest2.20240524.zip)
I moved the sky up, added playerclip "ceiling", added some sky laterally. (I looked at how other Jan's maps were made).
Maybe it might be further polished by added the lateral sky also as curved sky above the curved walls (at the moment I added the lateral sky above the lateral brushes which actually seal the map). I might already bee enough this way, though.

Further changes included in my test version:
- Changed color of the lower light "bars" behind the bases to match the color they emit. (#70)
- Moved two item_health entities (those which were the sides of the map, at the lightning gun) in order to fix the issue of bots repeatedly going back and forth between the two in a sort of infinite loop. Bots seem to roam better now. (#66)
- Added some small bonuses (small health and armor, machinegun ammo and item_botroam) to the lateral tunnels, in order to have bots use them some more. (#21)

Test with /devmap oacmpctf1_gigtest2.

I haven't used it to replace the version in the repository. I'd be glad if you wanted to use it and continue fixing the map from there.
